tabular/src/autogluon/tabular/models/tabular_nn/tabular_nn_quantile.py

```diff
@@ -451,3 +447,63 @@ def load(cls, path: str, reset_paths=True, verbose=True):
             model._architecture_desc = None
             model.model = torch.load(model.path + model.params_file_name)
         return model
+
+    def _hyperparameter_tune(self, X, y, X_val, y_val, scheduler_options, **kwargs):
```

Comment:
Do we really need this `_hyperparameter_tune` method? Or can the AbstractModel one be used for this model as well? Note that we've transitioned the TabularNeuralNet model to just use the AbstractModel `hyperparameter_tune` (so the data-loaders are recreated in each HPO trial, but this extra computation is pretty negligible compared to the NN training and it simplifies the code a lot).

Reply:
oh it's cool. I just removed and tested it, and it seems working properly, good!